yaml.go

```diff
@@ -0,0 +1,650 @@
+package llblib
+
+import (
+	"context"
+	"fmt"
+	"sort"
+	"strconv"
+	"time"
+
+	"github.com/coryb/walky"
+	"github.com/moby/buildkit/client/llb"
+	"github.com/moby/buildkit/solver/pb"
+	"github.com/opencontainers/go-digest"
+	"github.com/pkg/errors"
+	"golang.org/x/exp/maps"
+	"gopkg.in/yaml.v3"
+)
+
+// ToYAML will serialize the llb.States to sequence of yaml nodes.  If there
```

Comment:
Incomplete comment

Reply:
Thanks, no idea what I started to write there...